doc/api/child_process.markdown

```diff
@@ -408,11 +428,7 @@ const child = spawn('prg', [], {
 child.unref();
 ```
 
-When using the `detached` option to start a long-running process, the process
-will not stay running in the background after the parent exits unless it is
-provided with a `stdio` configuration that is not connected to the parent.
-If the parent's `stdio` is inherited, the child will remain attached to the
-controlling terminal.
+*Note: that detaching process works highly platform specific*
```

Comment:
Perhaps: `*Note that detaching a process is platform specific.*` ... also would be good to list the platform specific differences here.